### ISSUE TITLE
Javascript error on General settings page in Area settings in v1.7.0

### ISSUE THREAD

**Issue by bobvmierlo** [nielsfaber/alarmo#270]
### Checklist
- [X] I checked for similar existing issues (both open and closed) before posting.
- [X] I am running the latest version.

### Actual behavior
Javascript error on the General settings page at the Area settings of Alarmo after updating to v1.7.0

Error shown at the Area part:
```javascript
a=>{if(!(a instanceof bt))throw new Error("unsafeHTML can only be used in text bindings");const t=wt.get(a);if(void 0!==t&&ft(e)&&e===t.value&&a.value===t.fragment)return;const i=document.createElement("template");i.innerHTML=e;const s=document.importNode(i.content,!0);a.setValue(s),wt.set(a,{value:e,fragment:s})}
```

Console log shows this:
```
8eef858a.js:524 [Violation] Added non-passive event listener to a scroll-blocking 'touchstart' event. Consider marking event handler as 'passive' to make the page more responsive. See https://www.chromestatus.com/feature/5745543795965952
ready @ 8eef858a.js:524
ready @ 8eef858a.js:524
ready @ 8eef858a.js:5263
_enableProperties @ app.4e133b2d.js:5232
connectedCallback @ app.4e133b2d.js:5232
connectedCallback @ app.4e133b2d.js:5232
connectedCallback @ 8eef858a.js:5263
k @ alarmo:31
$ @ alarmo:31
_ @ alarmo:31
I @ alarmo:31
(anonymous) @ alarmo:37
update @ alarmo:37
performUpdate @ alarmo:25
Πq @ alarmo:25
```

Screenshot:
![image](https://user-images.githubusercontent.com/38190383/129473874-f11c68c3-ece5-4e97-b481-2ccec5fd7c97.png)

**Comment by nielsfaber** [comment]
Woops, that looks ugly.
Looks like I didn't do enough testing before releasing, I will work on a quick-fix for this.

**Comment by nielsfaber** [comment]
I made a v1.7.0a release which should take care of this.
Sorry for the trouble.

**Comment by bobvmierlo** [comment]
Updated to v1.7.0a with HACS and now it gives me this error in the HA errorlog:
```
Logger: homeassistant.setup
Source: setup.py:173
First occurred: 12:18:16 (1 occurrences)
Last logged: 12:18:16

Setup failed for alarmo: Integration not found.
```

**Comment by bobvmierlo** [comment]
After a full uninstall and reinstall the Integration also does not show up in HA to be added.

**Comment by nielsfaber** [comment]
I have no idea about that error, seems completely unrelated.
Maybe you updated through HACS before the release was completed (including  zip file with the files)?
Please check what is stored in `/custom_components/alarmo`

**Comment by bobvmierlo** [comment]
This is definitely a v1.7.0a related issue. When reverting back to 1.7.0 or 1.6.2 the integration works perfectly fine

**Comment by tteck** [comment]
I'm thinking maybe a versioning issue v1.7.0a vs v1.7.1?

**Comment by nielsfaber** [comment]
I am not aware of any rules regarding version numbering.
I just thought the change was so minimal i just wanted to append a letter, but maybe it was the wrong thought.
v1.7.0a is now replaced by v1.7.1, better to be safe than sorry.
Wouldn't know what else could be behind this, so hopefully this fixes the problem.

**Comment by tteck** [comment]
Fixed!

**Comment by bobvmierlo** [comment]
> Fixed!

Yep fixed here to

**Comment by tteck** [comment]
Is the original issue fixed?

**Comment by nielsfaber** [comment]
Great!
I will add this to the (long) list of lessons learned in working with Home Assistant.
Sorry for the troubles once again 😅 

I think this issue can be closed, as v1.7.1 should fix the original problem for which this issue was opened.

**Comment by bobvmierlo** [comment]
I can confirm the original issue to be fixed. Allthough I had to check in an incognito screen, somehow I can't get the cache to clear.
